### ISSUE TITLE
.NET Error on Linux

### ISSUE THREAD

**Issue by Meecr0b** [Orange-OpenSource/keepass-vault-sync-plugin#1]
Hi,
I got the following error message while loading the plugin:
```
The plugin cannot be loaded.
A newer .NET Framework is required
```
Are there any updates on [this discussion](https://sourceforge.net/p/keepass/discussion/329221/thread/cc951c6e/)?

**Comment by Karreg** [comment]
This is done, you were right to ask, I forgot about it...
Next release will allow loading on linux (and it works of course). However, make sure to have the 'mono-complete' package installed. You can try with 'mono-devel' but I had issues with only that package...

**Comment by Karreg** [comment]
Fixed in release 1.0.3
